docs/src/site/markdown/download.md

```diff
@@ -12,144 +12,54 @@
+See release specific installation instructions in [Documentation](documentation.html).
 
-**Example**
+* Server installation is described in the Trafodion Provisioning Guide
+* Client installation is described in the Trafodion Client Installation Guide
 
-```
-mkdir $HOME/trafodion
-mkdir $HOME/trafodion/downloads
-cd $HOME/trafodion/downloads
-wget <log4c++ RPM>
-wget <Trafodion Installer>
-wget <Trafodion Server>
-```
+To build Trafodion from source code, see the [Trafodion Contributor Guide](https://cwiki.apache.org/confluence/display/TRAFODION/Trafodion+Contributor+Guide).
 
-## Install log4c++
-
-Use **`yum install`** to install the `log4c++` RPM on all nodes on all nodes in your Hadoop cluster.
-
-## Unpack Trafodion Installer
-
-Use **`tar -zxf`** to unpack the Trafodion Installer.
-
-**Example**
-
-```
-cd $HOME/trafodion/downloads
-tar -zxf apache-trafodion-installer-1.3.0-incubating-bin.tar.gz -C $HOME/trafodion
-```
-
-## Create Configuration File
-
-The Trafodion Installer uses a configuration file that describes your Hadoop environment.
-
-**Copy Configuration Template File**
-
-```
-cd $HOME/trafodion/installer
-cp trafodion_config_default my_config
-```
-
-**Edit Configuration File**
-
-Use the embedded guidance in the configuration file to define each of these settings:
-
-|-------------------|----------------------------------------------|-------------------|--------------------------------|--------------------------------
-**`LOCAL_WORKDIR`** | **`TRAF_USER_PASSWORD`** (change if desired) | **`NODE_LIST`**   | **`node_count`**               | **`MY_NODES`**
-**`JAVA_HOME`**     | **`TRAF_PACKAGE`**                           | **`HADOOP_TYPE`** | **`URL`**                      | **`ADMIN`**
-**`PASSWORD`**      | **`CLUSTER_NAME`**                           | **`SQ_ROOT`**     | **`START`** (set to **`"Y"`**) | **`INIT_TRAFODION`** (set to **`"Y"`**)
-
-## Run `trafodion_install`
-
-You'll use the Trafodion Installer (**`trafodion_install`**) to create the Trafodion Runtime User (**`trafodion`**), 
-modify your system and Hadoop environment, restart ZooKeeper, HDFS, and HBAse to activate configuration changes, 
-and install/starts Trafodion.
-
-**Example**
+# Download
 
-```
-cd $HOME/trafodion/installer
-./trafodion_install --accept_license --config_file my_config
-```
+## 2.0.1
 
-The Trafodion Installer performs all the operations required to install and start Trafodion. Wait for it to complete before
-continuing to the next step.
+* Coming Soon
 
-## Verify Installation
+* [Release Notes](release-notes-2-0-1.html)
+* [Documentation](documentation.html#20x_Releases)
 
-Perform this step as the **`trafodion`** user.
+## 2.0.0
 
-First, check the status of the Trafodion environment.
+* [Release Notes](release-notes-2-0-0.html)
+* [Source Code Release][src200]  -  [PGP][pgp200] [MD5][md5200] [SHA1][sha200]
+* Convenience Binaries
```

Comment:
Should this mention that there is no client package in 2.0, so people don't search for it unnecessarily? Maybe something like a short note: Release 2.0.0 does not include convenience binaries for the ODBC/JDBC clients. Those must be built from source, if needed.